packages/enzyme-adapter-react-16.3/src/ReactSixteenThreeAdapter.js

```diff
@@ -483,6 +484,11 @@ class ReactSixteenThreeAdapter extends EnzymeAdapter {
     return typeOfNode(fragment) === Fragment;
   }
 
+  isCustomComponentElement(inst) {
+    const isFunctionOrForwardRef = () => typeof inst.type === 'function' || isForwardRef(inst);
```

Comment:
i think these can be inlined; no need to create a function for it

Reply:
done, the result is a little more confusing but it's faster :+1: